neo/Consensus/ConsensusService.cs

```diff
@@ -470,6 +470,12 @@ protected override void OnReceive(object message)
             }
         }
 
+        private void RequestRecovery()
+        {
+            if (context.BlockIndex == Blockchain.Singleton.HeaderHeight + 1)
+                localNode.Tell(new LocalNode.SendDirectly { Inventory = context.MakeRecoveryRequest() });
```

Comment:
I think this is `RespondWithRecovery` (directly sending a Recovery message to recover someone), not `RequestRecovery`.

Reply:
just to clarify.. for V=0 (or -1), the objective is to request help.. but on changeview, I thought it was providing recovery at the same time it was asking. But I think that jyst asking on both cases should be fine. Its much easier to understand using this logic, congratulations.